src/fsi/console.fs

```diff
@@ -112,13 +180,12 @@ type internal Cursor =
             Console.CursorLeft <- left)
 
     static member Move(inset, delta) =
-        let position =
-            Console.CursorTop * (Console.BufferWidth - inset)
-            + (Console.CursorLeft - inset)
-            + delta
+        ignore inset
```

Comment:
Can we just remove the `inset` argument from the method?

Reply:
yes.